thalium/icebox
# Tests minor changes

- Split win10 tests and main launcher of tests
- Add a doc file for tests
- Avoid an access violation memory if setup did not work properly (ASSERT instead of EXPECT)
- Remove redundant resume in attach test (already in TearDown)
- add common methods to be able to set a timeout on any function, running the VM between a lapse of time and test if we are paused in user mode
- add break conditions (with ASSERT) in processes test of win10 to avoid invalid accesses
